src/Core/Client/Client.php

```diff
@@ -1292,75 +1292,75 @@ public function createCoreAdmin(array $options = null): CoreAdminQuery
     }
 
     /**
-     * Create a managed resources query instance.
+     * Create a Collections API query instance.
```

Comment:
These methods were moved around to keep the `QueryType\Server` methods grouped together.